src/components/FormSelectionContainer/FormSelect/FormSelect.tsx

```diff
@@ -14,38 +15,80 @@ interface FormSelectProps {
 export const FormSelect: FunctionComponent<FormSelectProps> = ({ id, form, onAddForm, ...props }) => {
   const { config } = useConfigContext();
   const [errorMsg, setErrorMsg] = useState<string | null>(null);
-  const [isValidDns, setIsValidDns] = useState<boolean>();
+  const [isValidDns, setIsValidDns] = useState<boolean>(true);
+  const [isValidOwner, setIsValidOwner] = useState<boolean>(true);
+  const [isValidEntry, setIsValidEntry] = useState<boolean>(true);
```

Comment:
the default assumes all are valid at first load 🤔

Reply:
cos need wait for the call response on the page
if we are sticking with onclick, can default it to false